src/main/java/de/dennisguse/opentracks/fragments/StatisticsRecordingFragment.java

```diff
@@ -65,13 +65,10 @@ public static Fragment newInstance() {
             preferenceMetricUnits = PreferencesUtils.isMetricUnits();
         }
 
-        if (PreferencesUtils.isKey(R.string.stats_custom_layout_fields_key, key)) {
+        if (PreferencesUtils.isKey(R.string.stats_custom_layouts_key, key) || PreferencesUtils.isKey(R.string.stats_custom_layout_selected_profile_key, key)) {
```

Comment:
What is the difference between those two keys?

Reply:
`stats_custom_layouts_key`: it contains all layouts in a CSV way.
`stats_custom_layout_selected_layout_key`: it contains last selected layout in a CSV way.

Every time the user select a layout then it's copied to `stats_custom_layout_selected_layout_key` that will be use when user starts a new recording.